packages/engine/Source/Shaders/Voxels/IntersectionUtils.glsl

```diff
@@ -20,6 +20,36 @@ struct RayShapeIntersection {
     vec4 exit;
 };
 
+vec4 intersectionMin(in vec4 intersect0, in vec4 intersect1)
+{
+    if (intersect0.w == NO_HIT) {
```

Comment:
Do we need the same conditional for `intersect1`?

Reply:
The other conditional wasn't happening in current usage, but I agree, handling both cases would be more future-proof.